content/en/real_user_monitoring/browser/troubleshooting.md

```diff
@@ -41,7 +41,16 @@ You can also check your browser developer tools console or network tab if you no
 
 ### Data to the Datadog intake
 
-The RUM Browser SDK sends batches of data periodically to the Datadog intake. If data is being sent, you should see network requests targeting `/v1/input` (the URL origin part may differ due to RUM configuration) in the Network section of your browser developer tools:
+The RUM Browser SDK sends batches of data periodically to the Datadog intake.
```

Comment:
**🥜 nitpick:** ‏do we need to keep this first sentence? There is a bit of repetition with the next one.

Reply:
Agreed, I think we can remove this sentence.

Reply:
Removed, thanks for the catch!